client/src/components/InputField/InputField.js

```diff
@@ -72,6 +145,11 @@ InputField.propTypes = {
   type: PropTypes.string,
   autoFocus: PropTypes.bool,
   attributes: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+  tip: PropTypes.shape({
+    content: PropTypes.string.isRequired,
+    importance: PropTypes.string,
```

Comment:
If the list of possible values is known up front, then important should be a oneOf condition. Although I suspect we may want to add more possible values down the road.

Reply:
Definitely - I'm using the `TIP_IMPORTANCE_LEVELS` constant with oneOf to restrict this prop now.